### ISSUE TITLE
normalize pixels?

### ISSUE THREAD

**Issue by AIBluefisher** [googleinterns/IBRNet#15]
Hi,

I read the code in projection.py. However, I did not get why we need to normalize the pixels after reprojecting 3D points to the image plane.
```py
    def normalize(self, pixel_locations, h, w):
        resize_factor = torch.tensor([w-1., h-1.]).to(pixel_locations.device)[None, None, :]
        normalized_pixel_locations = 2 * pixel_locations / resize_factor - 1.  # [n_views, n_points, 2]
        return normalized_pixel_locations
```

**Comment by xavihart** [comment]
i think it is used for querying feature. 

```
feat_sampled = F.grid_sample(featmaps, normalized_pixel_locations, align_corners=True)
```
(ibrnet/projection.py:121)

**Comment by BlarkLee** [comment]
why multiply pixel_locations with 2?
